sources/packages/api/src/route-controllers/application/application.system.controller.ts

```diff
@@ -28,4 +28,32 @@ export class ApplicationSystemController {
 
     return { data: application.data };
   }
+
+  @Patch(":applicationId/assessment")
+  async updateAssessmentInApplication(
+    @Body() assessment: ApplicationAssessmentDTO,
+    @Param("applicationId") applicationId: number,
+  ): Promise<number> {
+    const application = await this.getByApplicationId(applicationId);
+    const updateAssessmentInApplication = await this.applicationService.updateAssessmentInApplication(
+      applicationId,
+      application,
+      assessment,
+    );
+    return updateAssessmentInApplication.affected;
+  }
+
+  @Get(":applicationId/assessment")
```

Comment:
Will this method be consumed by Camunda?
It seems that this would be a method being consumed by Vue application. If yes, please move to the correct controller with the proper authorization.

Reply:
yeah, I will move it